libs/application/templates/financial-aid/src/fields/Files/Files.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from 'react'
+import { InputFileUpload, UploadFile } from '@island.is/island-ui/core'
+
+import { useIntl } from 'react-intl'
+import { filesText } from '../../lib/messages'
+import { FileUploadContainer } from '..'
+import { useFileUpload } from '../../lib/useFileUpload'
+import { UploadFileType } from '../../lib/types'
+import { useFormContext } from 'react-hook-form'
+
+interface Props {
+  uploadFiles: UploadFile[]
+  fileKey: UploadFileType
+  folderId: string
+  hasError?: boolean
+}
+
+const Files = ({ uploadFiles, fileKey, folderId, hasError = false }: Props) => {
+  const { formatMessage } = useIntl()
+  const { setValue } = useFormContext()
+
+  const {
+    files,
+    uploadErrorMessage,
+    onChange,
+    onRemove,
+    onRetry,
+  } = useFileUpload(uploadFiles, folderId)
+
+  const stringifyFile = (file: UploadFile) => {
```

Comment:
Is there a better name for this function? Feel like stringifyFile is saying that it will turn the file into a string 🤔

Reply:
Do you have any suggestions? We named it this because this is doing the same as json.stringify, however It is not possible to directly convert a File object into JSON using JSON.stringify. So this function manually puts in the information of the file we need

Reply:
But is this function not returning an object, unlike json.stringify that returns a string?

Reply:
Well this is basically what the function does, what name do you think will suit best? 😇 
![Screenshot 2022-03-01 at 16 18 09](https://user-images.githubusercontent.com/62568905/156338128-d6d5ea48-7d87-4542-843b-5b266ef712b6.png)